FastSimulation/SimplifiedGeometryPropagator/python/GeometryESProducer_cfi.py

```diff
@@ -0,0 +1,16 @@
+import FWCore.ParameterSet.Config as cms
+
+# With an ideal geometry (for the reconstruction)
+from FastSimulation.TrackerSetup.TrackerMaterial_cfi import *
+GeometryESProducer = cms.ESProducer("GeometryESProducer",
```

Comment:
Please change the module name to be ore descriptive. Say `FastSimGeometryESProducer`.

Reply:
Hi, unfortunately, I don't know too much about this ESProducer business. Do we actually want an ESProducer here? I realized that only the python file exists and the actual plugin is missing in my code (there was no crash since the config file wasn't even imported anywhere). Should I write a ESProducer like the one in the old FastSim Propagator package which is not used anymore by the new framework?
https://github.com/cms-sw/cmssw/blob/master/FastSimulation/TrackerSetup/plugins/TrackerInteractionGeometryESProducer.cc

Reply:
Done.